app/controllers/branches_controller.rb

```diff
@@ -28,15 +29,17 @@ def edit
 
 	def update
 		if @branch.update(branch_params)
-			redirect_to @branch, notice: "Successfully updated branch"
+			redirect_to @branch
+			flash[:notice] = "Successfully updated branch"
```

Comment:
Yeah, that is user preference.  I regularly see it both ways.  Whatever works best for you.